input/pagecontent/reference.md

```diff
@@ -2343,7 +2344,7 @@ The [CodeableReference](https://hl7.org/fhir/2020Feb/references.html#codeableref
   // restrict the CodeableConcept aspect to a code in the LatexCodeVS value set:
   * code from LatexCodeVS (required)
   // restrict Reference aspect to an instance of SubstanceDefinition conforming to the LatexSubstanceDefinition profile:
-  * code only Reference(LatexSubstanceDefinition)
+  * code only CodeableReference(LatexSubstanceDefinition)
```

Comment:
Note that on line 2333 above it says:
> Making use of CodeableReference involves no new FSH syntax.

Now that we have the new `CodeableReference` keyword for type constraints, this sentence is a little too broad. Since we're in the section on assignments, I think we should change this sentence to be more specific:
> Assigning CodeableReference values involves no new FSH syntax.

Or we can just remove the sentence entirely. I'm fine either way.

Reply:
I think I like the idea of removing the sentence.

Reply:
I removed the sentence in [f3be4f8](https://github.com/HL7/fhir-shorthand/pull/159/commits/f3be4f84ccbd40ecb37e16b5474e41dbb020681a), but let me know if you think it needs further clarification.